packages/cli/src/cog/__test__/quadkey.vrt.test.ts

```diff
@@ -14,191 +17,272 @@ o.spec('quadkey.vrt', () => {
     const logger = LogConfig.get();
     LogConfig.disable();
 
+    const testDir = `${__dirname}/../../../__test.assets__`;
+
+    const googleProj = ProjectionTileMatrixSet.get(EpsgCode.Google);
+
+    function writePaddingQuadKey(quadKey = '31', proj = googleProj): void {
+        const tile = QuadKey.toTile(quadKey);
+        const qkBounds = proj.tileToSourceBounds(tile);
+        const px = proj.tms.pixelScale(tile.z);
+        const qkPadded = qkBounds.scaleFromCenter((qkBounds.width + px * 10) / qkBounds.width);
+
+        const { toWsg84 } = proj;
+
+        writeFileSync(
+            process.env.HOME + '/tmp/poly.geojson',
```

Comment:
I doubt most people have a `~/tmp`

Reply:
Only used for debugging not invoked in the code. If someone wants to use it they can create a tmp dir

Reply:
I think every other test like this just jams it in the current directory, you need to change this too as its causing a LGTM error, easiest way is just to export the function.

Reply:
I'll just rip it out for now